src/Pages/Search/Search.js

```diff
@@ -1,2 +1,299 @@
-import React from "react";
-import ReactDOM from "react-dom";
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
+import { FaSearch } from "react-icons/fa";
+
+export default function Search() {
+  const [categories, setCategories] = useState([]);
+
+  useEffect(() => {
+    fetch("http://localhost:3000/data/categoryList.json")
```

Comment:
```suggestion
    fetch("/data/categoryList.json")
```
로컬의 루트 경로를 활용할 경우 이렇게 사용할 수도 있습니다. 이 경우 길이도 짧고 포트번호에 구애받지 않는다는 장점이 있겠네요.

Reply:
오 처음 알았어요! 감사합니다 :)